db/db.js

```diff
@@ -5,11 +5,11 @@ const uri = process.env.MONGODB_URI
 
 //To-do retry every 10 seconds
 async function dbConnect() {
-    // console.log(uri)
+    console.log(uri)
     mongoose.connection.on('connected', () => console.log('connected'));
     mongoose.connection.on('open', () => console.log('open'));
     await mongoose.connect(uri, {
-        dbName: 'tw-db'
+        dbName: 'sarpheab-db'
```

Comment:
We will have new issue to move db name to ENV, so now we can just stick to tw-db